integrationtest/mapper-pojo/src/test/java/org/hibernate/search/v6poc/integrationtest/mapper/pojo/JavaBeanAnnotationMappingDiscoveryIT.java

```diff
@@ -6,8 +6,6 @@
  */
 package org.hibernate.search.v6poc.integrationtest.mapper.pojo;
```

Comment:
s/definines/defines/ in the commit message

Reply:
Done